2022-02-24 21:02:58,566 INFO    MainThread:21315 [wandb_setup.py:_flush():71] setting env: {}
2022-02-24 21:02:58,566 INFO    MainThread:21315 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-24 21:02:58,566 INFO    MainThread:21315 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220224_210258-2mf0ekyy/logs/debug.log
2022-02-24 21:02:58,566 INFO    MainThread:21315 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220224_210258-2mf0ekyy/logs/debug-internal.log
2022-02-24 21:02:58,566 INFO    MainThread:21315 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f715365cb10>
2022-02-24 21:02:58,567 INFO    MainThread:21315 [wandb_init.py:init():404] calling init triggers
2022-02-24 21:02:58,567 INFO    MainThread:21315 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-24 21:02:58,567 INFO    MainThread:21315 [wandb_init.py:init():460] starting backend
2022-02-24 21:02:58,567 INFO    MainThread:21315 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-24 21:02:58,661 INFO    MainThread:21315 [backend.py:ensure_launched():216] starting backend process...
2022-02-24 21:02:58,737 INFO    MainThread:21315 [backend.py:ensure_launched():222] started backend process with pid: 21368
2022-02-24 21:02:58,741 INFO    MainThread:21315 [wandb_init.py:init():469] backend started and connected
2022-02-24 21:02:58,811 INFO    MainThread:21315 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-24 21:02:58,812 INFO    MainThread:21315 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-24 21:02:58,812 INFO    MainThread:21315 [wandb_init.py:init():533] updated telemetry
2022-02-24 21:02:58,959 INFO    MainThread:21315 [wandb_init.py:init():563] communicating current version
2022-02-24 21:03:00,845 INFO    MainThread:21315 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-24 21:03:00,847 INFO    MainThread:21315 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-24 21:03:01,065 INFO    MainThread:21315 [wandb_init.py:init():606] starting run threads in backend
2022-02-24 21:03:06,077 INFO    MainThread:21315 [wandb_run.py:_console_start():1810] atexit reg
2022-02-24 21:03:06,079 INFO    MainThread:21315 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-24 21:03:06,079 INFO    MainThread:21315 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-24 21:03:06,081 INFO    MainThread:21315 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-24 21:03:06,081 INFO    MainThread:21315 [wandb_init.py:init():633] run started, returning control to user process
2022-02-24 21:03:17,621 INFO    MainThread:21315 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-24 21:03:17,621 INFO    MainThread:21315 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-24 21:06:14,032 INFO    MainThread:21315 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-24 21:06:14,033 INFO    MainThread:21315 [wandb_run.py:_restore():1752] restore
2022-02-24 21:06:14,286 INFO    MainThread:21315 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-24 21:06:14,508 INFO    MainThread:21315 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-24 21:06:15,154 INFO    MainThread:21315 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25163
}

2022-02-24 21:06:15,266 INFO    MainThread:21315 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25163
}

2022-02-24 21:06:15,381 INFO    MainThread:21315 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25163
}

2022-02-24 21:06:15,485 INFO    MainThread:21315 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25163
  total_bytes: 25163
}

2022-02-24 21:06:15,589 INFO    MainThread:21315 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25163
  total_bytes: 25163
}

2022-02-24 21:06:15,692 INFO    MainThread:21315 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25163
  total_bytes: 25163
}

2022-02-24 21:06:15,796 INFO    MainThread:21315 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25163
  total_bytes: 25163
}

2022-02-24 21:06:15,899 INFO    MainThread:21315 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25163
  total_bytes: 25163
}

2022-02-24 21:06:16,002 INFO    MainThread:21315 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25163
  total_bytes: 25163
}

